ts/features/euCovidCert/components/EuCovidCertHeader.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { View } from "native-base";
+import { WithEUCovidCertificateHeaderData } from "../types/EUCovidCertificate";
+import { H1 } from "../../../components/core/typography/H1";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+import { H2 } from "../../../components/core/typography/H2";
+import { isStringNullyOrEmpty } from "../../../utils/strings";
+
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: "row",
+    justifyContent: "space-between"
+  },
+  column: { flexDirection: "column", flex: 1 },
+  logo: {
+    width: 84,
+    height: 84
+  }
+});
+
+/**
+ * header with
+ * | title       | logo |
+ * | subtitle    |      |
+ * @param props
+ * @constructor
+ */
+export const EuCovidCertHeader = (props: WithEUCovidCertificateHeaderData) => (
+  <>
+    <View style={styles.row}>
+      <View style={styles.column}>
+        <H1 style={IOStyles.flex} testID={"EuCovidCertHeaderTitle"}>
```

Comment:
```suggestion
        <H1 testID={"EuCovidCertHeaderTitle"}>
```
I propose to remove the `flex` style in order to avoid the excessive spacing between title and subtitle and use the natural line heigth of the textual component.

**Before:**
<img width="389" alt="Schermata 2022-01-19 alle 18 09 53" src="https://user-images.githubusercontent.com/26501317/150179935-f08175a3-9e7c-40bc-aef1-095f63a51eb7.png">



**After:**
<img width="400" alt="Schermata 2022-01-19 alle 18 09 29" src="https://user-images.githubusercontent.com/26501317/150179872-fb1cd794-dd0c-4aca-bb56-a4370a64f14b.png">

Do you think could be an improvement?

Reply:
🦅 
https://github.com/pagopa/io-app/pull/3666/commits/58f12c621b33c41dd3948419305ef31fb2f5ba55